reqs/models.py

```diff
@@ -31,18 +31,18 @@ def tags_for(cls, model, instance=None, **extra_filters):
 
 @unique
 class PolicyTypes(Enum):
-    memorandum = 'Memorandum'
     circular = 'Circular'
-    strategy = 'Strategy'
-    review = 'Policy Review'
     executive = 'Executive Order'
     guidance = 'Guidance'
-    standards = 'Standards'
+    hspd = 'HSPD'
     law = 'Law'
+    memorandum = 'Memorandum'
+    national_action = 'National Action Plan'
     plan = 'Plan'
-    hspd = 'HSPD'
     ppd = 'PPD'
-    national_action = 'National Action Plan'
+    review = 'Policy Review'
+    standards = 'Standards'
+    strategy = 'Strategy'
```

Comment:
Heads up that re-ordering these will likely require a Django migration -- the database won't change, but Django likes to serialize the full configuration of models (including the order of options).